server/src/main/java/org/elasticsearch/search/fetch/FetchSubPhaseProcessor.java

```diff
@@ -28,4 +31,18 @@
      */
     void process(HitContext hitContext) throws IOException;
 
+    /**
+     * Called when profiling after processing all documents to get any extra
+     * debug information the phase collected.
+     */
+    default Map<String, Object> getDebugInfo() {
+        return null;
+    }
+
+    /**
+     * Profiles from child-phases.
+     */
+    default List<ProfileResult> childProfiles() {
```

Comment:
No current subprocessor overrides this, could we remove until a follow-up where we implement profiling for inner_hits subphases?